javascript/javascript1/week1/age-ify.js

```diff
@@ -0,0 +1,7 @@
+//Age-ify
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+
+let age = yearFuture - yearOfBirth;
```

Comment:
Since this is never reassigned, you can also declare this as a `const`.

Reply:
Yes- I have changed it to const